src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,154 @@
+package duke;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    private ArrayList<Task> lst;
+
+    /**
+     * Constructor for Tasklist class
+     */
+    public TaskList(){
+        lst = new ArrayList<>();
+    }
+
+    /**
+     * method to retrieve the arraylist in a tasklist
```

Comment:
The Javadocs here could be improved upon to better adhere to the standards provided. An example can be found [here](https://nus-cs2103-ay2122s1.github.io/website/book/documentation/tools/javaDoc/what/), where the description for functions start with verbs such as "Returns" or "Calculates". The Javadocs of other methods has similar areas for improvement, so perhaps it might be better if they were checked?

Reply:
I agree with mweeruien, in this case perhaps it could be 
```suggestion
     * Getter method to retrieve the arraylist in a tasklist. 
     or 
     * Returns the ArrayList object stored in a tasklist. 
```
Remember to capitalise the first word and add commas to end of sentences.